### PR TITLE
Documentation revamp

### PR DESCRIPTION
**Pull request by NicoZelaya**
https://splitio.atlassian.net/browse/SDKS-911

### CODE REVIEW

File: CONTRIBUTORS-GUIDE.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+6. Run all tests and make sure there are no failures.
+7. `git push` your changes to GitHub within your topic branch.
+8. Open a Pull Request(PR) from your forked repo and into the `development` branch of the original repository.
+9. When creating your PR, please fill out all the fields of the PR template, as applicable, for the project.
+10. Check for conflicts once the pull request is created to make sure your PR can be merged cleanly into `development`.
+11. Keep an eye out for any feedback or comments from Split's SDK team.
+  
+### Building the SDK
+   
+Javi: ADD CONTENT HERE (Instructions on how to build this library, as simple as possible)
```
Comment (NicoZelaya) [inline CONTRIBUTORS-GUIDE.md]: @javrudsky please complete this section and remove line 23.

File: CONTRIBUTORS-GUIDE.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+   
+### Running tests
+   
+Javi: ADD CONTENT HERE (Instructions on how to run automated tests as simple as possible)
```
Comment (NicoZelaya) [inline CONTRIBUTORS-GUIDE.md]: @javrudsky please complete this section and remove line 27

File: CONTRIBUTORS-GUIDE.md [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+  
+### Linting and other useful checks
+  
+Javi: ADD CONTENT HERE (Instructions on how to run Swiftlint, as well as any other checks you may have available. as simple as possible)
```
Comment (NicoZelaya) [inline CONTRIBUTORS-GUIDE.md]: @javrudsky please complete this section and remove line 31

File: README.md [hunk 4]
```diff
@@ -1,43 +1,56 @@
...
+This SDK is compatible with iOS deployment target versions 9.0 and later, and Swift version 4 and later.
+
+## Getting started
+Below is a simple example that describes the instantiation and most basic usage of our SDK:
+```
+<Factory instantiation>
+<Block until Ready>
+<Get treatment>
+<If treatment == ...>
+```
```
Comment (NicoZelaya) [inline README.md]: @javrudsky please complete this section and remove placeholder tags. This example would be as simple as possible, minimal code to get a treatment and use it on the ifs. 
See for guidance:
https://help.split.io/hc/en-us/articles/360020401491-iOS-SDK#2-instantiate-the-sdk-and-create-a-new-split-client
https://help.split.io/hc/en-us/articles/360020401491-iOS-SDK#basic-use

File: CONTRIBUTORS-GUIDE.md [hunk 5]
```diff
@@ -0,0 +1,68 @@
...
+        if treatment == "on" {
+            print("I'm ON")
+        } else if treatment == "off" {
+            print("I'm OFF")
+        } else {
+            print("CONTROL was returned, there was an error")
+        }
+    }
+}
+```
```
Comment (EmilianoSanchez) [inline CONTRIBUTORS-GUIDE.md]: This basic usage example should be moved to README.md, in the placeholder: 
```
<Factory instantiation>
<Block until Ready>
<Get treatment>
<If treatment == ...>
```

**Pull request merged**